### ISSUE TITLE
Extract scalars & histogram plots as image(download instantly)

### ISSUE THREAD

**Issue by soylisK** [tensorflow/tensorboard#2658]
A useful feature:
I would like to see the option of extracting accuracy or loss or variables(like weights, etc) plots directly from tensorboard and save it as an image file, for use in presentations and staff like that.

 We need to save time and invest in the actual development of our neural nets!

**Comment by stephanwlee** [comment]
Not true for histograms but you can download the scalars as an image today.

![image](https://user-images.githubusercontent.com/2547313/64968416-f9b9c800-d856-11e9-89ce-7c6a04292663.png)

**Comment by soylisK** [comment]
In what version? I am using 1.9.0 and there is only the option of extract as CSV or JSON at least for scalar that I've tested recently.

**Comment by stephanwlee** [comment]
@soylisK, it is available as of 1.12: https://github.com/tensorflow/tensorboard/releases/tag/1.12.0
Is there a reason why you are using the older version?

**Comment by soylisK** [comment]
I am working on a project which started by using 1.9 one year+ ago, I didn't want to upd in order to avoid any trouble! Anyway since there is, at a newer version I think I have to close the issue for this feature

**Comment by DanielWicz** [comment]
It doesn't work with histograms (can't download, version 2.9)
